### ISSUE TITLE
Replication Wildcard issue for MySQL

### ISSUE THREAD

**Issue by flarco** [slingdata-io/sling-cli#16]
The following replication YAML does not work.

```yaml
source: MYSQL_SOURCE
target: MYSQL_TARGET

# default config options which apply to all streams
defaults:
  mode: full-refresh 

  # specify pattern to use for object naming in target connection, see below for options
  object: 'new_db.{stream_schema}_{stream_table}'

streams:
  "`my-db-name`.*": {}
```
The table name is rendered as `"my-db-name"."table_name"`, when it should be using backticks.

Furthermore, when using object formating, such as `object: 'new_db.{stream_schema}_{stream_table}'`, the `{stream_schema}` does not replace the `-` from `my-db-name`. Should be `my_db_name`.

**Comment by azngeek** [comment]
What would be a suitable workaround? 

As the `object-pattern` seems to be mandatory. I tried the following configuration:
`object: 'new_db'.{stream_table}`. Maybe the object-pattern could be optional and will try to replicate the original structure if no configuration is give.

**Comment by flarco** [comment]
No, you don't have to use the pattern syntax. You can specify the object name for each stream:

```yaml
source: MYSQL_SOURCE
target: MYSQL_TARGET

# default config options which apply to all streams
defaults:
  mode: full-refresh 

  # specify pattern to use for object naming in target connection, see below for options
  object: 'new_db.{stream_schema}_{stream_table}'

streams:
  '`my-db-name`.my_table1': 
      object: new_db.my_table1
  '`my-db-name`.my_table2': 
      object: new_db.my_table2
      mode: incremental
      primary_key: [id] 
```

**Comment by azngeek** [comment]
I understand. So instead of the wildcard, I will then specify a new entry for each `table/stream` until the issue will be resolved.

**Comment by flarco** [comment]
OK, should be fixed in newly released `v1.0.38`. It will keep the hyphen in the end, but will properly qualify the object name. Wildcards should also work now.
